fastlane/metadata/i18ncli.js

```diff
@@ -0,0 +1,320 @@
+    // Remove a potentially existing directory with the same name
+    try {
+      await fsp.rmdir( newDirectoryPath, { recursive: true } );
+      console.log( "Removed existing directory", newDirectoryPath );
+    } catch ( e ) {
+      // Directory did not exist
+    }
+    console.log( "Renaming directory", directoryPath, "to", newDirectoryPath );
+    await fsp.rename( directoryPath, newDirectoryPath );
+  } ) );
+  const androidDirectories = await fsp.readdir( path.join( __dirname, "android" ), {
+    withFileTypes: true
+  } ).then( files => files.filter( file => file.isDirectory( ) ).map( file => file.name ) );
+  console.log( "Current list of the directories in fastlane/metadata/android", androidDirectories );
+  return Promise.all( androidDirectories.map( async directory => {
+    const locale = ANDROID_MAPPINGS[directory];
+    if ( !locale ) return;
+    const directoryPath = path.join( __dirname, "android", directory );
+    const newDirectoryPath = path.join( __dirname, "android", locale );
+    // Remove a potentially existing directory with the same name
+    try {
+      await fsp.rmdir( newDirectoryPath, { recursive: true } );
+      console.log( "Removed existing directory", newDirectoryPath );
+    } catch ( e ) {
+      // Directory did not exist
+    }
+    console.log( "Renaming directory", directoryPath, "to", newDirectoryPath );
+    await fsp.rename( directoryPath, newDirectoryPath );
+  } ) );
+}
+
+async function removeUnsupportedDirectories( ) {
+  // Get all directories in fastlane/metadata/ios
+  const iosDirectories = await fsp.readdir( path.join( __dirname, "ios" ), {
+    withFileTypes: true
+  } ).then( files => files.filter( file => file.isDirectory( ) ).map( file => file.name ) );
+  console.log( "Current list of the directories in fastlane/metadata/ios", iosDirectories );
+  await Promise.all( iosDirectories.map( async directory => {
+    if ( SUPPORTED_IOS_METADATA_LOCALES.indexOf( directory ) >= 0 ) return;
+    const directoryPath = path.join( __dirname, "ios", directory );
+    console.log( "Removing unsupported directory", directoryPath );
+    await fsp.rmdir( directoryPath, { recursive: true } );
+  } ) );
+  const androidDirectories = await fsp.readdir( path.join( __dirname, "android" ), {
+    withFileTypes: true
+  } ).then( files => files.filter( file => file.isDirectory( ) ).map( file => file.name ) );
+  console.log( "Current list of the directories in fastlane/metadata/android", androidDirectories );
+  return Promise.all( androidDirectories.map( async directory => {
+    if ( SUPPORTED_ANDROID_METADATA_LOCALES.indexOf( directory ) >= 0 ) {
+      console.log( "Supported directory", directory );
+      return;
+    }
+    const directoryPath = path.join( __dirname, "android", directory );
+    console.log( "Removing unsupported directory", directoryPath );
+    await fsp.rmdir( directoryPath, { recursive: true } );
+  } ) );
+}
+
+async function copyAndroidTitle() {
```

Comment:
It took me a sec to understand that this exists because we don't wan the title to be localized, so we copy the English title to directories for all locales. It would help to add a comment to that effect here.

Reply:
Added a comment